README.md

```diff
@@ -97,6 +97,29 @@ client := goshopify.NewClient(app, "shopname", "")
 // Fetch the number of products.
 numProducts, err := client.Product.Count(nil)
 ```
+### Client Options
+When creating a client there are configuration options you can pass to NewClient. Simply use the last variadic param and 
+pass in the built in options or create your own and manipulate the client. See [options.go](https://github.com/bold-commerce/go-shopify/blob/master/options.go)
+for more details.
+
+#### WithVersion
+Read more details on the [Shopify API Versioning](https://shopify.dev/concepts/about-apis/versioning)
+to understand the format and release schedules. You can use `WithVersion` to specify a specific version 
+of the API. If you do not use this option you will be defaulted to the latest stable API.
```

Comment:
I believe it defaults to the oldest stable version if not set (according to shopify documentation)
> If your request doesn't include a version, then the API also defaults to the oldest supported stable version.

Please could you fix it?

Reply:
yes of course!

Reply:
fixed!